airbyte-integrations/connectors/destination-bigquery/src/test-integration/kotlin/io/airbyte/integrations/destination/bigquery/BigqueryWriteTest.kt

```diff
@@ -129,25 +149,53 @@ class StandardInsertRawOverride :
         CdcDeletionMode.HARD_DELETE,
     )
 
+class StandardInsertRawOverrideProto :
+    BigqueryDirectLoadWriteTest(
+        BigQueryDestinationTestUtils.standardInsertRawOverrideConfig,
+        CdcDeletionMode.HARD_DELETE,
+        isStreamSchemaRetroactiveForUnknownTypeToString = false,
```

Comment:
why are we behaving differently in proto mode? shouldn't that be purely an implementation detail?

Reply:
In proto mode UnknownType should not be a possibility. If a column type is unknown then we can query it from the proto record. We need to use the right method call to get the data. If we dont know the type, we cant use the method call.

Reply:
probably worth adding a comment explaining why we need this